lib/system/jssys.nim

```diff
@@ -105,6 +105,11 @@ proc rawWriteStackTrace(): string =
   else:
     result = "No stack traceback available\n"
 
+proc writeStackTrace() =
+  var trace = rawWriteStackTrace()
+  trace.setLen(trace.len - 1)
+  echo trace
```

Comment:
nit: `trace.setLen(trace.len - 1)` is fragile wrt future code evolution, instead check if last element if `\n` first

nit: doc comment says `stderr` so should have a note for js 

the best IMO would be to allow `stderr.write` for js so it would automatically work for all client code calling `stderr.write`
* js with nodejs would use stderr.writeln
* js without -d:nodejs would use `console.error(...)` (not sure how to handle `\n`)
* vm/nimscript code would use a vmops